includes/agentic-commerce/class-wc-stripe-agentic-commerce-inventory-tracker.php

```
@@ -238,4 +293,139 @@ public function sync_inventory(): void {
+				[
+					'archives'      => count( $pending ),
+					'import_set_id' => $result['import_set_id'] ?? null,
+					'status'        => $result['status'] ?? 'unknown',
+				]
+			);
+
+			// Clear pending archives on success.
+			delete_option( self::PENDING_ARCHIVES_OPTION );
+
+			// Clean up the temporary file.
+			$file_path = $feed->get_file_path();
```

PHPMD (2.15.0)

[error] The variable $file_path is not named in camelCase. (undefined)

(CamelCaseVariableName)